index.js

```diff
@@ -781,6 +781,12 @@ Feed.prototype.has = function (start, end) {
   return total === this.bitfield.total(start, end)
 }
 
+Feed.prototype.head = function (opts, cb) {
+  if (typeof opts === 'function') return this.head({}, opts)
+  if (this.length === 0) cb(new Error('feed is empty'))
+  else this.get(this.length - 1, cb)
```

Comment:
Do you think it would be confusing to have `.get` return immediately but `.head` to wait for `ready`?